### PR TITLE
Get mocks3 on par with moto

### PR DESCRIPTION
**Pull request by jordanvance**
Bringing mocks3 into the 2020s with compatibility with moto 2.2.20.

### CODE REVIEW

**Review by strichter — APPROVED** [review]

**Comment by strichter** [comment]
When doing a release, this should probably become version 2.0.0 given that the APIs changed quite a bit.

File: src/shoobx/mocks3/tests/test_s3_boto3.py [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
 #
 ###############################################################################
 """Shoobx S3 Backend
 """
-from __future__ import unicode_literals
 
 import boto3
 import functools
-import mock
+from unittest import mock
```
Comment (strichter) [inline src/shoobx/mocks3/tests/test_s3_boto3.py:11]: Move to other `from...` statements.

**Comment by miromad** [comment]
LGTM

**Review by miromad — COMMENTED** [review]
LGTM

**Pull request merged**